rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/41_date_range.yml

```diff
@@ -0,0 +1,178 @@
+setup:
+  - do:
+      indices.create:
+          index: test
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              properties:
+                date:
+                  type: date
+                  format: epoch_second
+
+  - do:
+      cluster.health:
+        wait_for_status: green
+
+---
+"Date range":
+  - do:
+      index:
+        index: test
+        id: 1
+        body: { "date" : 1000 }
+
+  - do:
+      index:
+        index: test
+        id: 2
+        body: { "date" : 2000 }
+
+  - do:
+      index:
+        index: test
+        id: 3
+        body: { "date" : 3000 }
+
+  - do:
+      indices.refresh: {}
```

Comment:
It'd be a little more readable to do this with `bulk` and a single refresh. Maybe even a little faster.